packages/traefik-config-scripts/spec.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
haha no, I'm just happy to see that you've separated the two packages. No change needed. :)